queries/runescript/highlights.scm

```diff
@@ -0,0 +1,92 @@
+; Variables
+[
+  (identifier)
+  (local_variable)
+  (game_variable)
+  (constant_variable)
+  (parameter_variable)
+  (declaration_variable)
+] @variable
+
+; Scripts
+(script
+  trigger: (identifier) @keyword.directive
+  subject: (identifier) @function)
+
+; Calls
+(call
+  [
+    "~"
+    "@"
+  ]? @function
+  callee: (identifier) @function)
+
+; Literals
+((literal) @constant.builtin
+  (#any-of? @constant.builtin "true" "false" "null"))
+
+[
+  (integer_literal)
+  (coord_literal)
```

Comment:
Is that an integer or a float?

Reply:
Neither, it's a special literal type that represents a map coordinate, it's 5 integers separated by underscores.

Reply:
Hmm, then it's not really a number.

Reply:
I'd still like to keep it as such, if you don't mind. This is pretty much how it's done in the existing non-treesitter syntax highlighting, and we'd like to keep those closely matched.

Reply:
Sorry, but this is a non-goal for the queries in this repo -- it's much more important to be consistent across the different languages, so colorschemes can target the official standard captures without (too many) language-specific special cases.

@ribru17 ?

Reply:
Not sure; `@string.special` possibly, which is what is used elsewhere in similar (but not the same) situations.

Reply:
Alright, let's go with that one then.

Reply:
Maybe if the grammar exposes child nodes as `(integer)` `"_"`  ... in the future we can highlight the numbers as numbers and then `@punctuation.delimiter` for underscores

Reply:
> Maybe if the grammar exposes child nodes as (integer) "_" ... in the future we can highlight the numbers as numbers and then @punctuation.delimiter for underscores

That's a good idea for future iterations, but I think I will keep it as @string.special for now (until we change the parsing a little bit).